### ISSUE TITLE
allowPrivate doesn't seem to be working

### ISSUE THREAD

**Issue by EvanHahn** [gajus/eslint-plugin-jsdoc#275]
I may be doing something wrong here, but `allowPrivate` doesn't seem to be working for me.

My source file looks like this:

```js
/**
 * @private
 */
function square (n) {
  return n * n
}

module.exports = square
```

My `.eslintrc.json` looks like this:

```js
{
  "env": {
    "node": true
  },
  "extends": ["eslint:recommended"],
  "plugins": ["jsdoc"],
  "settings": {
    "jsdoc": {
      "allowPrivate": true
    }
  },
  "rules": {
    "jsdoc/require-returns": "error"
  }
}
```

But I still get the `require-returns` error.

```
error  Missing JSDoc @returns declaration  jsdoc/require-return
```

I've tried setting `allowPrivate` to `false` and it has the exact same behavior.

I've put a sample repo in [a gist that you can use to reproduce this](https://gist.github.com/EvanHahn/383939ec948c29eaa878429656ba97a8).

Again, I may be doing something wrong here. Thanks!

**Comment by brettz9** [comment]
Apologies, this was misdocumented (fixed in `master`). The setting should be `ignorePrivate`. That should fix it, so closing, but feel free to comment further/reopen if any issues.
